recipe/meta.yaml

```diff
@@ -33,21 +33,20 @@ requirements:
     - pyzmq >=23.0
     - tornado >=6.0
```

Comment:
`tornado` needs to be at least `6.2`: https://github.com/jupyter/jupyter_client/blob/main/pyproject.toml#L32

Reply:
Fixed, thanks!